src/shared/Core.Tests/Commands/GetCommandTests.cs

```diff
@@ -16,14 +17,21 @@ public async Task GetCommand_ExecuteAsync_CallsHostProviderAndWritesCredential()
         {
             const string testUserName = "john.doe";
             const string testPassword = "letmein123"; // [SuppressMessage("Microsoft.Security", "CS001:SecretInline", Justification="Fake credential")]
-            ICredential testCredential = new GitCredential(testUserName, testPassword);
+            const string testRefreshToken = "xyzzy";
+            const long testExpiry = 1919539847;
+            ICredential testCredential = new GitCredential(testUserName, testPassword) {
+                OAuthRefreshToken = testRefreshToken,
+                PasswordExpiry = DateTimeOffset.FromUnixTimeSeconds(testExpiry),
+            };
             var stdin = $"protocol=http\nhost=example.com\n\n";
             var expectedStdOutDict = new Dictionary<string, string>
             {
                 ["protocol"] = "http",
                 ["host"]     = "example.com",
                 ["username"] = testUserName,
-                ["password"] = testPassword
+                ["password"] = testPassword,
+                ["password_expiry_utc"] = testExpiry.ToString(),
+                ["oauth_refresh_token"] = testRefreshToken,
```

Comment:
The `refresh_token` will likely also have an expiry date.
At least one observed implementation however does not report the constraint.